pkg/driver/efs_watch_dog.go

```diff
@@ -26,9 +26,17 @@ import (
 	"k8s.io/klog"
 )
 
-// https://github.com/aws/efs-utils/blob/v1.28.2/dist/efs-utils.conf
+// https://github.com/aws/efs-utils/blob/v1.30.2/dist/efs-utils.conf
```

Comment:
This is the v1.30.2 config file, any reason we don't keep the consistency with the version in DockerFile?

Reply:
Planning to bump efs-utils to 1.30.2 when it is available in linux repo. There isn't too much difference between conf files for 1.30.1 & 1.30.2 so decided to copy over the content of 1.30.2